pipelines/internal/simple-signing-pipeline/simple-signing-pipeline.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: simple-signing-pipeline
   labels:
-    app.kubernetes.io/version: "0.3.0"
+    app.kubernetes.io/version: "0.3.1"
```

Comment:
This is a big change, actually a breaking change, since the parameter names changed, so should be `1.0.0`.